# The trigger to pick up an object
---
